deploy/operator.yaml

```diff
@@ -40,7 +40,7 @@ spec:
         - name: managed-upgrade-operator
           # Replace this with the built image name
           # This will get replaced on deploy by /hack/generate-operator-bundle.py
-          image: quay.io/app-sre/managed-upgrade-operator:latest
+          image: quay.io/app-sre/managed-upgrade-operator:v0.1.809-46d986a
```

Comment:
(this comment applies to this line as well as the one in `csv-template.yaml`)

I think one of the concerns that led to the original ticket was that having a valid version set here means that there is the possibility that - if some pipelines happen to not replace the image correctly and also didn't fail - MUO could be deployed running an old version and no-one would notice unless they looked at it.

So setting it to a real version still leaves that risk in place. It should be a invalid version such that it _should_ cause errors if somehow this deployment were go out, which would highlight a failure in the deployment pipeline processes.

Reply:
I've had a go at pushing this change to the branch.